_posts/2019-12-18-road-to-scala-3.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: blog-detail
+post-type: blog
+by: Lukas Rytz, Adriaan Moors, Martin Odersky
+title: "Scala 2 Roadmap Update: the Road to Scala 3"
+---
+
+Together with the Scala 3 team at EPFL (aka the Dotty team), led by Martin Odersky, we have decided that, rather than developing Scala 2.14, our efforts should go to Scala 3 instead.
```

Comment:
You should say who is "we"/"our" in that sentence. ;)

Reply:
It would be hard to list exhaustively who all was involved in this decision, but I see this as a joint statement of a joint decision by the authors of the post.

Reply:
I had Seb's reaction as well, and the problem gets worse later in the sentence with “our efforts”. IIRC a previous draft of this post said “the Scala team at Lightbend” so I'd suggest putting that back in in some form. As it stands, it is puzzling and reads strangely, and it especially matters when it's the first sentence of the whole post.

The problem continues in the following paragraphs, too, where phrases like "we're very excited to shift our focus to Scala 3, we will continue to maintain Scala 2.13" are clearly about the Lightbend team, but the post never said so.

I do appreciate the desire to avoid wording that implies that the Lightbend team unilaterally made this decision, which of course isn't the case.

How about:

> We are excited and pleased to announce that the Scala team at Lightbend will join the Scala 3 effort, rather than work on Scala 2.14. The team, led by Adriaan and Lukas, will join forces with the Dotty team at EPFL, led by Martin, to ensure that Scala 3.0 is a timely and high quality release.

(no strong feeling about including "led by Adriaan and Lukas", but it might help clarity?)

Reply:
There's a lot to say, and I agree the sentences aren't the most beautiful, but rewording these things again would require internal approval at Lightbend, which given timezones and full calendars would likely push the post to the other side of the holiday shutdown. I'd rather go with the suboptimal wording and get something out now.

Reply:
Okay, understood.